### ISSUE TITLE
Upgrade to v3.1.0: "Your bundle is locked to httplog (1.4.1)"

### ISSUE THREAD

**Issue by clement-analogue** [tootsuite/mastodon#13058]
I'm trying to upgrade Mastodon to v3.1.0, however, at the step `bundle install`, I have the following error
```
$ bundle install
Fetching gem metadata from https://rubygems.org/............
Your bundle is locked to httplog (1.4.1), but that version could not be found in any of the sources listed in your Gemfile. If you haven't changed sources, that means the author of httplog (1.4.1) has removed it. You'll need to update your bundle to a version other than httplog (1.4.1) that hasn't been removed in order to install.
```

I am running Mastodon on Ubuntu 18.04.

**Comment by Findus23** [comment]
This looks like https://github.com/tootsuite/mastodon/issues/13057, which I just reported

**Comment by ThibG** [comment]
Fixed in 3.1.1
